### PR TITLE
[chromestyle] - manage admin template module style

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #17054 .

### Summary of Changes
manage admin template module style like "moduleposition"

### Testing Instructions
edit a admin module from backend >advandec tab
and check for Module Style

### Expected result
module style ar from admin templates
expected

### Actual result

module style ar from fronted templates

### CODE REVIEW

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 81816414b01e12c42a7cc5408f4038e2b90e610b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17057">issues.joomla.org/tracker/joomla-cms/17057</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 81816414b01e12c42a7cc5408f4038e2b90e610b<br /><br />Thanks for doing this so quickly<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17057">issues.joomla.org/tracker/joomla-cms/17057</a>.</sub>

File: libraries/cms/form/field/chromestyle.php [hunk 1]
```diff
@@ -27,6 +27,103 @@ class JFormFieldChromeStyle extends JFormFieldGroupedList
...
+	 *
+	 * @var    integer
+	 * @since  3.2
+	 */
+	protected $clientId;
+
+	/**
+	 * Method to get certain otherwise inaccessible properties from the form field object.
+	 *
+	 * @param   string  $name  The property name for which to the the value.
```
Comment (Quy) [inline libraries/cms/form/field/chromestyle.php]: Remove duplicate `the the`. Also the sentence doesn't read right.

File: libraries/cms/form/field/chromestyle.php [hunk 2]
```diff
@@ -27,6 +27,103 @@ class JFormFieldChromeStyle extends JFormFieldGroupedList
...
+				return $this->clientId;
+		}
+
+		return parent::__get($name);
+	}
+
+	/**
+	 * Method to set certain otherwise inaccessible properties of the form field object.
+	 *
+	 * @param   string  $name   The property name for which to the the value.
```
Comment (Quy) [inline libraries/cms/form/field/chromestyle.php]: Remove duplicate `the the`. Also the sentence doesn't read right.

**Comment by ghost** [comment]
RTC after two successful tests.

File: libraries/cms/form/field/chromestyle.php (hunk 1 above)
Comment (alikon) [reply libraries/cms/form/field/chromestyle.php]: i hope it is more readable now

**Pull request merged**